src/bindings/python/src/pyopenvino/frontend/frontend.cpp

```diff
@@ -22,15 +22,16 @@ void regclass_frontend_FrontEnd(py::module m) {
 
     fem.def(
         "load",
-        [](FrontEnd& self, const std::string& s) {
-            return self.load(s);
+        [](FrontEnd& self, const py::object& path) {
+            const std::string path_str = path.str();
```

Comment:
Is there a way to confirm that either `string` or `Path` object is passed that way? It may be necessary to restrict it to these types, as many things can utilize `__str__` attribute.